packages/circuit-ui/components/Modal/Modal.tsx

```diff
@@ -57,6 +66,7 @@ export const Modal = forwardRef<HTMLDialogElement, ModalProps>((props, ref) => {
     hideCloseButton,
     variant = 'contextual',
     className,
+    preventClose,
```

Comment:
Let's be explicit about the default value here in case the underlying default values in Dialog change.

```suggestion
    preventClose = false,
```